corrections to tips (#24)
matplotlib/cheatsheets: handout-tips.tex

% -----------------------------------------------------------------------------
\subsection*{\rmfamily Rasterization}

> If your figure is made of a lot of graphical elements such as a huge

Reviewer: ```suggestion
If your figure has many graphical elements such as a huge
```

> scatter, you can rasterize them to save memory and keep other elements

Reviewer: ```suggestion
scatter or pcolormesh, you can rasterize them to save memory and keep other elements
```

\end{tabular}

\subsection*{\rmfamily Colorbar adjustment}

> You can adjust a colorbar's aspect ratio when adding it.

Reviewer: ```suggestion
You can adjust a colorbar's size when adding it.
```

This example doesn't actually change the aspect ratio?
